tICA/scripts/feature_helpers/findScaledMADOutliers.m

```diff
@@ -0,0 +1,16 @@
+function isOutlier = findScaledMADOutliers(data)
+% Determine if an array contains values that are outside of three scaled MAD
+    numElems = numel(data);
+    isOutlier = false(size(data));
```

Comment:
This line isn't needed.

Also, I prefer not to use variable names that are very similar to names of matlab-provided functions (like `isoutlier`), due to the matlab syntax for function call and variable indexing being ambiguous (even worse, something like `rs = ls;` can call the function `ls()` if there is no variable with that name...).

Reply:
that's...interesting